driver-extras/pom.xml

```diff
@@ -142,18 +142,39 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>animal-sniffer-maven-plugin</artifactId>
-                <configuration>
-                    <!--
-                    There is no support for JDK 8 yet, see
-                    https://github.com/mojohaus/animal-sniffer/issues/1
-                    -->
-                    <skip>true</skip>
-                    <signature>
-                        <groupId>org.codehaus.mojo.signature</groupId>
-                        <artifactId>java18</artifactId>
-                        <version>1.0</version>
-                    </signature>
-                </configuration>
+                <executions>
+                    <execution>
+                        <id>check</id>
+                        <phase>process-classes</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <signature>
+                                <groupId>org.codehaus.mojo.signature</groupId>
+                                <artifactId>java16</artifactId>
+                                <version>1.0</version>
+                            </signature>
+                            <annotations>
+                                <annotation>com.datastax.driver.extras.codecs.jdk8.IgnoreJDK6Requirement</annotation>
+                            </annotations>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>check-jdk8</id>
```

Comment:
One thing that tripped me up was why the `check-jdk8` execution?   My guess is this is for future-proofing if we ever do jdk9 specific stuff? stuff.